<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>images</class>
 <widget class="QDialog" name="images">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setup Seeds</string>
  </property>
  <widget class="mousePos" name="imgShow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>118</width>
     <height>19</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>IMAGE TO SEED</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="boxSeeds">
        <property name="title">
         <string>Seeds</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="rbBG">
             <property name="text">
              <string>BG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDone">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="mouseLocation">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mousePos</class>
   <extends>QLabel</extends>
   <header location="global">mousepos.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
